schemas/access/oidc-permission-1.yml

```diff
@@ -28,6 +28,15 @@ properties:
             type: string
             enum:
             - /vault-config/policy-1.yml
+  instance:
+    "$ref": "/common-1.json#/definitions/crossref"
+    "$schemaRef":
+      type: object
+      properties:
+        '$schema':
+          type: string
+          enum:
+          - /vault-config/instance-1.yml
```

Comment:
what's wrong with:
```
"$ref": "/common-1.json#/definitions/crossref"
"$schemaRef": "/vault-config/instance-1.yml"
```

Reply:
Not a thing. Copied a different ref example thinking it was a standard approach. Changed references to simpler form